.gitignore

```diff
@@ -24,3 +24,4 @@ public/js/
 /.phpstorm.meta.php
 /phpstan.neon
 /.phpunit.result.cache
+app-modules/**/vendor
```

Comment:
For the purpose of the modules being individual units, we should have this within each module and add it to the scaffolding generated with the creation of each new module.